usaspending_api/submissions/serializers.py

```diff
@@ -7,4 +7,6 @@ class SubmissionAttributesSerializer(LimitableSerializer):
     class Meta:
 
         model = SubmissionAttributes
-        fields = '__all__'
+        fields = (
```

Comment:
This will now mean that users can't acquire any of the fields other than the ones expressly denoted here. Is this what we want?